chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/listingvalidation/ListingValidationReportEntity.java

```diff
@@ -0,0 +1,68 @@
+package gov.healthit.chpl.scheduler.job.listingvalidation;
+
+import java.util.Date;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "listing_validation_report")
```

Comment:
This line means we need a data-model PR, right?

Reply:
For got to PR it.  Should be there now..